packages/ckb-utils/types/sha3/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'sha3' {
+  export const SHA3: any
+  export const Keccak: any
```

Comment:
Is it possible to not declare `Keccak` type since we won't use it? (Just a question. If so this would avoid some future confusion.)

Reply:
To be completed and published to DefinitelyTyped